pkgs/development/compilers/gcc/builder.sh

```diff
@@ -251,7 +251,7 @@ postInstall() {
     fi
 
     if type "install_name_tool"; then
-        for i in "${!outputLib}"/lib/*.*.dylib; do
+        for i in "${!outputLib}"/lib/*.*.dylib "${!outputLib}"/lib/*.so.[0-9]; do
```

Comment:
Is it a bug in gcc buildsystem that they install libgccjit as .so file?

Reply:
Looks like this needs to be fixed here: https://github.com/gcc-mirror/gcc/blob/41d6b10e96a1de98e90a7c0378437c3255814b16/gcc/jit/Make-lang.in#L43

Reply:
Yep, looks like it. They still make weird linux-soname based assumptions around how the structure works, but I think we could just set this as a make variable (or patch it) to fix. Probably the better way to fix it!

Reply:
They have it conditionals for windows but not for macOS now: https://github.com/gcc-mirror/gcc/blob/229752afe3156a3990dacaedb94c76846cebf132/gcc/jit/Make-lang.in#L48

Reply:
Anyway if you don't patch the buildsystem would it not be better to rename it from .so to .dylib?